packages/pluggableWidgets/sidebar-web/src/components/Sidebar.tsx

```diff
@@ -28,6 +39,23 @@ export function Sidebar(props: PropsWithChildren<SidebarProps>): ReactElement {
         width = props.width;
     }
 
+    useEffect(() => {
+        if (sidebarRef.current && props.collapsible) {
+            const content = sidebarRef.current?.querySelectorAll(CONTENT_SELECTOR);
+            if (expanded) {
+                content.forEach(contentNode => {
+                    contentNode.removeAttribute("aria-hidden");
+                    contentNode.setAttribute("data-focusindex", "0");
+                });
+            } else {
+                content.forEach(contentNode => {
+                    contentNode.setAttribute("aria-hidden", "true");
+                    contentNode.setAttribute("data-focusindex", "-1");
+                });
+            }
```

Comment:
Nit: Can you just do if inside the `forEach` callback

Reply:
Fixed